lib/erb_lint/cli.rb

```diff
@@ -169,6 +169,13 @@ def load_options(args)
       option_parser.parse!(args)
     end
 
+    def load_linters
+      @load_linters ||= begin
+        ERBLint::LinterRegistry.load_custom_linters
```

Comment:
Is there any reason why this is not being done at `ERBLint::LinterRegistry.linters`?

Reply:
`linters` is an `attr_reader` for an array with no logic at the moment. It's populated when files include the `LinterRegistry` class.

https://github.com/Shopify/erb-lint/blob/b1e657aae1951400059e344346937d3f062a0fc4/lib/erb_lint/linter_registry.rb#L7-L14

For linters that come with the project, this kinda all happens magically with autoload, but for linters defined outside of ERB Lint they need to be explicitly loaded by calling `load_custom_linters`:

https://github.com/Shopify/erb-lint/blob/b1e657aae1951400059e344346937d3f062a0fc4/lib/erb_lint/linter_registry.rb#L20-L23

I don't have any historical context as to why this is the case.

Reply:
It feels more logical have it there. Maybe the `attr_reader` should be called `loaded_linters`, and `linters` should call `load_custom_linters` before returning the value of the instance variable? That way we wouldn't have to change `ERBLint::CLI` at all.

Reply:
I like that approach! @jen-taylor-shopify and I will pair tomorrow and give it a whirl, thanks!